File: baehyeonwoo.works/HyeonWeb/SourceCodes/PublicSource/ServerListPingDesign/JVPaperServerListPingDesign/src/main/java/works/baehyeonwoo/paperserverlistpingdesign/PaperMain.java
package works.baehyeonwoo.paperserverlistpingdesign;

import org.bukkit.Bukkit;
import org.bukkit.plugin.java.JavaPlugin;
import works.baehyeonwoo.paperserverlistpingdesign.paperevent.PaperEvent;

public class PaperMain extends JavaPlugin {
    public void onEnable() {
        Bukkit.getPluginManager().registerEvents(new PaperEvent(), this);
    }
}

File: baehyeonwoo.works/HyeonWeb/SourceCodes/PublicSource/ServerListPingDesign/KTPaperServerListPingDesign/src/main/kotlin/works/baehyeonwoo/paperserverlistpingdesign/PaperMain.kt
package works.baehyeonwoo.paperserverlistpingdesign

import org.bukkit.Bukkit
import org.bukkit.plugin.java.JavaPlugin
import works.baehyeonwoo.paperserverlistpingdesign.paperevent.PaperEvent

/**
 *
 * @author BaeHyeonWoo
 *
 */


class PaperMain : JavaPlugin() {

    @Override
    override fun onEnable() {
        Bukkit.getPluginManager().registerEvents(PaperEvent(), this)
    }
}
